magenta/magenta/models/nsynth/ours/train.py

```diff
@@ -55,6 +56,8 @@ def main(unused_argv=None):
 
   logdir = FLAGS.logdir
   tf.logging.info("Saving to %s" % logdir)
+  train_files = glob.glob(FLAGS.train_path + "/*")
```

Comment:
Have you checked that this works even if FLAGS.train_path ends with a slash character?